### DISCUSSION TITLE
Top Level Await Error

### DISCUSSION THREAD

**Discussion by sjonas50** [vercel/next.js#46780] [Help]
### Summary

Im getting the following error:
./lib/sanity.client.ts
Module parse failed: The top-level-await experiment is not enabled (set experiments.topLevelAwait: true to enabled it)
Error: The top-level-await experiment is not enabled (set experiments.topLevelAwait: true to enabled it)

Here is my sanity.client.ts file:
export const projectId = process.env.NEXT_PUBLIC_SANITY_PROJECT_ID!;
export const dataset = process.env.NEXT_PUBLIC_SANITY_DATASET!;
const apiVersion = process.env.NEXT_PUBLIC_SANITY_API_VERSION!;

export const client = createClient({
    projectId,
    dataset,
    apiVersion,
    useCdn: false,
});

const data = await client.fetch(groq`*[]`)

And my next.config.js:
/** @type {import('next').NextConfig} */
module.exports = {
  reactStrictMode: true,
  experimental: {
    appDir: true
  }
}

Any idea how to fix this?

**Comment by guitheengineer** [comment]
Same issue as #43382 

Workaround is: 
```js
/** @type {import("next").NextConfig} */
module.exports = {
  experimental: { appDir: true },
  webpack(config) {
    config.experiments = { ...config.experiments, topLevelAwait: true }
    return config
  },
}
 ```
